SCons/Tool/ninja/ninja.xml

```diff
@@ -0,0 +1,293 @@
+            <item>NINJA_GENERATED_SOURCE_SUFFIXES</item>
+            <item>NINJA_MSVC_DEPS_PREFIX</item>
+            <item>NINJA_BUILDDIR</item>
+            <item>NINJA_REGENERATE_DEPS</item>
+            <item>NINJA_COMPDB_EXPAND</item>
+            <item>NINJA_ENV_VAR_CACHE</item>
+            <item>NINJA_POOL</item>
+            <item>DISABLE_AUTO_NINJA</item>
+            <item>__NINJA_NO</item>
+
+            <item>NINJA_FILE_NAME</item>
+
+            <item>NINJA_ALIAS_NAME</item>
+            <item>NINJA_SYNTAX</item>
+
+            <item>_NINJA_REGENERATE_DEPS_FUNC</item>
+        </sets>
+
+        <uses>
+            <item>CCCOM</item>
+            <item>CXXCOM</item>
+            <item>SHCXXCOM</item>
+            <item>CC</item>
+            <item>CXX</item>
+
+            <item>LINKCOM</item>
+            <item>LINK</item>
+
+            <item>SHLINKCOM</item>
+            <item>SHLINK</item>
+
+            <item>ARCOM</item>
+            <item>AR</item>
+            <item>RANLIB</item>
+            <item>ARFLAGS</item>
+            <item>RANLIBCOM</item>
+            <item>PLATFORM</item>
+            <item>ESCAPE</item>
+
+        </uses>
+    </tool>
+
+    <builder name="Ninja">
+        <summary>
+            <para>
+                &b-Ninja; is a special builder which
+                adds a target to create a ninja build file.
+                The builder does not require any source files to be specified,
+
+            </para>
+            <para>
+                If called with no arguments,
+                the builder will default to a target name of
+                <filename>ninja.build</filename>.
+            </para>
+            <para>
+                If called with a single positional argument,
+                &scons; will "deduce" the target name from that source
+                argument, giving it the same name, and then
+                ignore the source.
+                This is the usual way to call the builder if a
+                non-default target name is wanted.
+            </para>
+            <para>
+                If called with either the
+                <parameter>target=</parameter>
+                or <parameter>source=</parameter> keyword arguments,
+                the value of the argument is taken as the target name.
+                If called with both, the
+                <parameter>target=</parameter>
+                value is used and <parameter>source=</parameter> is ignored.
+                If called with multiple sources,
+                the source list will be ignored,
+                since there is no way to deduce what the intent was;
+                in this case the default target name will be used.
+            </para>
+            <note>
+                <para>
+                    You must load the &t-ninja; tool prior to specifying
+                    any part of your build or some source/output
+                    files will not show up in the compilation database.
```

Comment:
is this an unedited copy-paste from the compilationdb page?

Reply:
Docs are not ready for consumption mostly. So it could be.

Reply:
Fixed in next push.